napari/_qt/qt_viewer.py

```diff
@@ -262,9 +262,10 @@ def _on_cursor(self, event):
 
     def _on_reset_view(self, event):
         if isinstance(self.view.camera, ArcballCamera):
-            self.view.camera._quaternion = self.view.camera._quaternion.create_from_axis_angle(
+            quat = self.view.camera._quaternion.create_from_axis_angle(
                 *event.quaternion
             )
+            self.view.camera._quaternion = quat
```

Comment:
lol @ the concept of _you_ being "nosy" on a napari PR.
The issue there was `E501` (line too long).  Sometimes, black is unable to come up with a satisfying solution (like in that case with a few long chained attributes).  Black will fail gracefully but flake8 won't, so that was just a way to decrease the line length.

Reply:
Ah good! It hadn't occurred to me that black could be unable to fix that. Happy with this workaround. Thanks!

Reply:
Also: I try to be mindful to minimize annoying nitpicks in PRs, but sometimes I can't help myself! :joy: